composer.json

```
@@ -22,8 +22,8 @@
     "require-dev": {
         "phpro/grumphp": "^0.17.1 || ^0.19.0",
         "phpstan/phpstan": "^0.11.15 || ^0.12.0",
-        "phpunit/phpunit": ">=7.5",
-        "squizlabs/php_codesniffer": "^3.0",
+        "phpunit/phpunit": ">=9.0",
+        "squizlabs/php_codesniffer": "^3.5",
```

```suggestion
        "squizlabs/php_codesniffer": "^3.4",
```
This way it will maintain compatibility with antidot-fw/framework package. see https://github.com/antidot-framework/antidot-framework/blob/fd2a41645bc1ec1f6d1022be59134efd82a55112/composer.json#L31